spec/spec_helper.rb

```diff
@@ -105,3 +113,5 @@
   Kernel.srand config.seed
 =end
 end
+
+require "rails-patterns"
```

Comment:
@stevo I was getting `uninitialized constant Rails` when `require "rails-patterns"` was placed before `Rails` was declared (see my comment to your previous change request, I have explained it there), but it seems, that it works now 😮
I was apparently doing something wrong before (probably late night programming, after an exhausting day 😜), sorry for that confusion!